stable/influxdb/templates/NOTES.txt

```diff
@@ -0,0 +1,16 @@
+InfluxDB can be accessed via port {{ .Values.config.http.bind_address }} on the following DNS name from within your cluster:
```

Comment:
Can you include mention of the default username/password somewhere? I am trying to wire this chart up with the Chronograf one and I dont know what to enter on Chronograf.